### PR TITLE
Permits specifying a formatter in NLS.get(count)

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/kernel/nls/NLS.java [hunk 1]
```diff
@@ -313,7 +346,7 @@ public static String get(@Nonnull String property, int numeric, @Nullable String
             if (versions.length == 3) {
                 return versions[0].trim();
             } else {
-                return Formatter.create(versions[1].trim()).set("count", 0).format();
+                return Formatter.create(versions[1].trim()).set("count", formatter.apply(0)).format();
             }
         } else if (numeric == 1) {
```
Comment (jakobvogel) [inline src/main/java/sirius/kernel/nls/NLS.java:351]: Why do we format / support the "count" attribute for numeric 0, but not 1?

**Review by jakobvogel — APPROVED** [review]
Test case?

**Review by sabieber — APPROVED** [review]

File: src/main/java/sirius/kernel/nls/NLS.java (hunk 1 above)
Comment (andyHa) [reply src/main/java/sirius/kernel/nls/NLS.java:351]: Because this is usually a string from the NLS value e.g. "one" instead of "1"

Comment (andyHa) [reply src/main/java/sirius/kernel/nls/NLS.java:351]: We support 2-part patterns: One | Many -> in this case we format the 0 case like many or 3-part patterns: Zero | One | Many

**Pull request merged**